tests/location/UserLocation.test.ts

```diff
@@ -0,0 +1,90 @@
+import { LocationAccuracy } from "expo-location"
+import {
+  expoTrackUserLocation,
+  LocationCoordinatesMocks,
+  UserLocationTrackingAccurracy
+} from "@lib/location"
+
+const testAccuracy = "precise"
+
+describe("UserLocation tests", () => {
+  describe("ExpoUserLocationTracking tests", () => {
+    test("translates accurracy to proper expo accurracy when making tracking request", () => {
```

Comment:
It would be better not to need a unit test for mapping, js gives us plenty of options to have a mapping between strings (enums, objects)